analyzer/codechecker_analyzer/cmd/fixit.py

```diff
@@ -0,0 +1,349 @@
+        return True
+
+    def match_reports(diag_msg):
+        """
+        Check if the fixit belongs to a CodeChecker report from "reports"
+        object. In case "reports" are not given, this function returns True.
+        """
+        if not reports:
+            return True
+
+        row, col = get_location_by_offset(diag_msg['FilePath'],
+                                          int(diag_msg['FileOffset']))
+        return any(row == report['line'] and col == report['column'] and
+                   diag_msg['FilePath'] == report['checkedFile']
+                   for report in reports)
+
+    not_existing_files = set()
+    existing_files = set()
+    modified_files = set()
+
+    checker_names = make_regex(checker_names)
+    file_paths = make_regex(file_paths)
+
+    items = filter(
+        lambda diag: checker_names.match(diag['DiagnosticName']) and
+        len(diag['DiagnosticMessage']['Replacements']) != 0 and
+        file_paths.match(diag['DiagnosticMessage']['FilePath']) and
+        match_reports(diag['DiagnosticMessage']) and
+        check_existence_and_collect(diag['DiagnosticMessage']['FilePath']) and
+        check_modification_and_collect(diag['DiagnosticMessage']['FilePath']),
+        content['Diagnostics'])
+
+    if interactive:
+        items = filter(ask_user, items)
+
+    content['Diagnostics'] = list(items)
+
+    for diag in content['Diagnostics']:
+        existing_files.add(diag['DiagnosticMessage']['FilePath'])
+
+    return existing_files, not_existing_files, modified_files
+
+
+def list_fixits(inputs, checker_names, file_paths, interactive, reports):
+    """
+    This function dumps the .yaml files to the standard output like a "dry run"
+    with the replacements. See clang_tidy_fixit_filter() for the documentation
+    of the filter options.
+    inputs -- A list of report directories which contains the fixit dumps in a
+              subdirectory named "fixit".
+    """
+    not_existing_files = set()
+    existing_files = set()
+    modified_files = set()
+
+    for i in inputs:
+        fixit_dir = os.path.join(i, 'fixit')
+
+        if not os.path.isdir(fixit_dir):
+            LOG.info('No fixits in %s', i)
+            continue
+
+        for fixit_file in os.listdir(fixit_dir):
+            fixit_file = os.path.join(fixit_dir, fixit_file)
+            with open(fixit_file, encoding='utf-8', errors='ignore') as f:
+                content = yaml.load(f, Loader=yaml.BaseLoader)
+                fixit_mtime = util.get_last_mod_time(fixit_file)
+
+                existing, not_existing, modified = clang_tidy_fixit_filter(
+                    content, checker_names, file_paths, reports, fixit_mtime,
+                    interactive)
+
+                existing_files.update(existing)
+                not_existing_files.update(not_existing)
+                modified_files.update(modified)
+            print(yaml.dump(content))
+
+    if existing_files:
+        print("Updated files:\n{}".format(
```

Comment:
Why do you use print instead of LOG a couple of places?

And what do you mean under updated files? Files what the user changed? In this case maybe Changed files would be better.